ts/screens/authentication/TemporaryBlockedCie.tsx

```diff
@@ -0,0 +1,108 @@
+import { Option } from "fp-ts/lib/Option";
+import { Button, Container, H1, Text, View } from "native-base";
+import * as React from "react";
+import { StyleSheet } from "react-native";
+import { NavigationScreenProp, NavigationState } from "react-navigation";
+import { connect } from "react-redux";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+import FooterWithButtons from "../../components/ui/FooterWithButtons";
+import IconFont from "../../components/ui/IconFont";
+import I18n from "../../i18n";
+import customVariables from "../../theme/variables";
+
+type OwnProps = Readonly<{
+  navigation: NavigationScreenProp<NavigationState>;
+}>;
+type Props = OwnProps;
+type State = Readonly<{
+  securityCode: Option<string>;
+  holder: Option<string>;
+}>;
+const styles = StyleSheet.create({
+  contentContainerStyle: {
+    padding: customVariables.contentPadding
+  },
+  text: {
+    fontSize: customVariables.fontSizeBase
+  }
+});
+
+class TemporaryBlockedCie extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+  }
+
+  public render(): React.ReactNode {
+    const cancelButtonProps = {
+      block: true,
+      cancel: true,
+      onPress: (): boolean => this.props.navigation.goBack(),
+      title: I18n.t("global.buttons.cancel")
+    };
+    const retryButtonProps = {
+      block: true,
+      primary: true,
+      onPress: () => undefined,
```

Comment:
add a reference to the story that implements the condition here. You could also add an alert to display the message "NOT IMPLEMENTED":
```suggestion
      onPress: () => Alert.alert(I18n.t("global.notImplemented")),
```